### ISSUE TITLE
SwiftUI Preview Fails

### ISSUE THREAD

**Issue by peck** [apollographql/apollo-ios#1039]
I think I've followed the installation instructions on the docs page, https://www.apollographql.com/docs/ios/installation/, and my builds works and runs on the similar just fine, but my swiftui preview breaks with this error in the diagnostics. I'm new to swiftui (though fairly familiar with Apollo/graphql from other projects) and searching didn't find similar issue.

apollo-ios -> 0.23.0
Xcode -> 11.3.1

```
use of unresolved identifier \'Process\'

----------------------------------------

SchemeBuildError: Failed to build the scheme "practice"

use of unresolved identifier 'Process'

Compile /Users/peck/Library/Developer/Xcode/DerivedData/practice-ffshylstkbdpuwdygwokyrmihofw/SourcePackages/checkouts/apollo-ios/Sources/ApolloCodegenLib/Basher.swift:
/Users/peck/Library/Developer/Xcode/DerivedData/practice-ffshylstkbdpuwdygwokyrmihofw/SourcePackages/checkouts/apollo-ios/Sources/ApolloCodegenLib/Basher.swift:27:16: error: use of unresolved identifier 'Process'
    let task = Process()
               ^~~~~~~
```

On apollo-ios 0.22.0 I don't see the issue and preview seems to generally work as expected

**Comment by designatednerd** [comment]
Hi! What package manager are you using? Can you validate that the `ApolloCodegenLib` is not included in the libraries that are being added to your iOS application? It's designed to only be used with MacOS, though it seems like I should probably add some defensive `if` statements around that.

**Comment by designatednerd** [comment]
@peck The defensive `@available` statements have shipped with `0.23.1` which should prevent compilation of any of the classes that aren't available on iOS. I'm gonna close this out, please reopen if you're still seeing this issue when using 0.23.1. Thank you!

**Comment by peck** [comment]
Apologies for not seeing your first reply, didn’t get a notification from github for that one.

I’ll give the new version a try and see if it fixes it.

To answer the question, I’m using swift package manager and did not have codegen lib included.

Thanks!

**Comment by peck** [comment]
Following up, I tried with the latest release and get the same error

**Comment by iancanderson** [comment]
👋 I got the same error using `0.23.0`, and still in `0.23.1`. Downgrading to `0.22.0` fixed the issue for me as well - SwiftUI preview is working again

**Comment by designatednerd** [comment]
What the...oy. Yeah 0.22.0 doesn't have the `ApolloCodegenLib` at all so it would definitely not have this problem. 

Reopening - can you please take a look at the `Build Phases` for the target you're running SwiftUI with and make sure you do not have the `ApolloCodegenLib` in either your `Link Binary With Libraries` phase or your `Embed Frameworks` phase?

**Comment by TizianoCoroneo** [comment]
I had setup Apollo with SwiftPM, and I'm importing it as a dependency for another local package, so there is no Build phase to look at 😅 but I've found a different fix and opened a PR. 
https://github.com/apollographql/apollo-ios/pull/1066

**Comment by designatednerd** [comment]
Actual fix shipping with 0.23.2 courtesy @TizianoCoroneo
